src/ingest_validation_tools/directory-schemas/af.yaml

```diff
@@ -1,20 +1,20 @@
 LC Data:
-  '*': 0 # TODO: What is expected here?
+  '*': { required: False, description: "TODO" } # TODO: What is expected here?
 metadata:
   assay:
-    AF.csv: 1
+    AF.csv: { description: "TODO" }
 templates:
-  '*': 0 # TODO: What is expected here?
+  '*': { required: False, description: "TODO" } # TODO: What is expected here?
 VAN*-*K-*-*_*:
```

Comment:
As defined here, AF would only support *K*idney data from *VAN*derbilt. That is too restrictive.

Reply:
I've made an issue for Chris: #349